app/controllers/staff/activity_forms_controller.rb

```diff
@@ -16,6 +16,10 @@ class Staff::ActivityFormsController < Staff::BaseController
     :tied_status,
   ]
 
+  # This regex will match the three date params from `date_field`;
+  # e.g. actual_start_date(2i) actual_start_date(1i) actual_start_date(3i)
+  DATE_PARAM_REGEX = /.*_date\(.*/
```

Comment:
This is clever! When I see a comment I wonder if a nice, well named method would help slow-pokes like me figure out what this does?

Reply:
Done 👍